src/common/Smi.Common/Options/GlobalOptions.cs

```diff
@@ -273,10 +273,9 @@ public class DicomReprocessorOptions
 
         public ProducerOptions ReprocessingProducerOptions { get; set; }
 
-        public override string ToString()
-        {
-            return GlobalOptions.GenerateToString(this);
-        }
+        public TimeSpan SleepTime { get; set; }
+
+        public override string ToString() => GlobalOptions.GenerateToString(this);
```

Comment:
In future lets do this kind of thing (using statements, whitespace and => properties etc) in a seperate PR.  Makes it easier to view only the changes for the feature

Reply:
Sure, that makes sense. I think the issue is that I use the auto-formatter in VS which results in lots of diffs like that. I did investigate using [pre-commit](https://pre-commit.com/)/StyleCop before but didn't want to roll that out without chatting to everyone first.

Reply:
Yeah I get a bit of that too from visual studio but I just unstage the bits that are pointless (like whitespace / using etc) and don't bother to commit them.  Takes a bit more time but helps pick up accidental code changes etc too.